modules/cloudbuild/main.tf

```diff
@@ -138,9 +157,10 @@ resource "google_cloudbuild_trigger" "main_trigger" {
  ***********************************************/
 
 resource "google_cloudbuild_trigger" "non_main_trigger" {
-  for_each    = var.create_cloud_source_repos ? toset(var.cloud_source_repos) : []
-  project     = module.cloudbuild_project.project_id
-  description = "${each.value} - terraform plan."
+  for_each        = var.create_cloud_source_repos ? toset(var.cloud_source_repos) : []
+  project         = module.cloudbuild_project.project_id
+  description     = "${each.value} - terraform plan."
+  service_account = var.terraform_sa_name
```

Comment:
Have we tested pushing some TF config through this trigger? IIUC we also need to setup a bucket log and pass that in via the build file.

Reply:
I will do a deploy of the foundation using this version

Reply:
logging bucket and related permissions added